build/tasks/task-serve.js

```diff
@@ -25,8 +26,14 @@ gulp.task('serve', async () => {
     '--port', ports[0],
   ];
 
+  const runnerArgs = [
+    ...args,
+    ...(yargs.argv.browser ? ['--browser', yargs.argv.browser] : []),
```

Comment:
Main thinking with both of the comments about args (browser and mockOs) is that it's non-default behavior that we might not need and we could keep things simpler without adding it.  If you want the options now feel free to keep them.